src/app/components/Icons/Icons.tsx

```diff
@@ -26,24 +26,34 @@ type IconProps = {
   type: IconType;
   onClick?: () => void;
   className?: string;
+  isClickable: boolean;
 };
 
-const iconSizeMap: {
-  [type in IconType]: JSX.Element;
-} = {
-  calendar: <CalendarIcon />,
-  delete: <DeleteIcon />,
-  done: <DoneIcon />,
-  edit: <EditIcon />,
-  overview: <OverviewIcon />,
-  plus: <PlusIcon />,
-  plusBackground: <PlusBackgroundIcon />,
-  flowerSmall: <FlowerIconSmall />,
-  letsgo: <LetsGoIcon />,
-  division: <DivisionLineIcon />,
-};
+const Icons = ({ type, onClick, isClickable }: IconProps): JSX.Element => {
+  const currentPage = {
+    fill: 'var(--color-secondary)',
+    stroke: 'var(--color-secondary)',
+  };
+
+  const otherPage = {
+    fill: 'var(--color-action)',
+    stroke: 'var(--color-action)',
+  };
 
-const Icons = ({ type, onClick }: IconProps): JSX.Element => {
+  const iconSizeMap: {
+    [type in IconType]: JSX.Element;
+  } = {
+    calendar: <CalendarIcon {...(isClickable ? currentPage : otherPage)} />,
+    delete: <DeleteIcon />,
+    done: <DoneIcon />,
+    edit: <EditIcon />,
+    overview: <OverviewIcon {...(isClickable ? currentPage : otherPage)} />,
+    plus: <PlusIcon />,
+    plusBackground: <PlusBackgroundIcon />,
+    flowerSmall: <FlowerIconSmall />,
+    letsgo: <LetsGoIcon />,
+    division: <DivisionLineIcon />,
+  };
   return <div onClick={onClick}>{iconSizeMap[type]}</div>;
```

Comment:
It's rather unusual to give an onClick to a div. 
I think it would be better to either use a button instead of a div or give each icon the onClick prop.